examples/common/config-e2e.ts

```diff
@@ -0,0 +1,90 @@
+import { defineConfig, devices } from "@playwright/test";
+import type nodeProcess from "node:process";
+import { getAppPort, getInspectorPort, type AppName } from "./apps";
+
+declare const process: typeof nodeProcess;
+
+export function configurePlaywright(
+  app: AppName,
+  {
+    // Do we run on CI?
+    isCI = false,
```

Comment:
why is this an argument and not just computed here?
you always do pass it as `!!process.env.CI` so it would just make sense to do that directly in this file, wouldn't it?

Reply:
In general, I don't link that functions have external dependencies.
Injecting the value enable:
- writing tests,
- run on any platform (i.e. outside of GH actions)